doc/conf.py

```diff
@@ -63,7 +65,7 @@
 user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36 Edg/108.0.1462.54"
 
 try:
-    import sphinxcontrib.spelling
+    import sphinxcontrib.spelling # noqa: F401
```

Comment:
It might be good to add this to the dev guide as how to stop the linter from linting :)

Reply:
Done!